packages/yourloops/test/integration/pages/trends/trends-view-common.spec.tsx

```diff
@@ -81,15 +85,24 @@ describe('Trends view for anyone', () => {
       })
       expect(await screen.findByText('There is no CGM data for this time period :(')).toBeVisible()
     })
-  })
 
-  describe('with time in range data', () => {
-    it('should display correct readings in range stats info', async () => {
-      mockDataAPI(timeInRangeStatsTrendViewData)
+    it('should render correct tooltip and values GMI', async () => {
```

Comment:
Following up with my previous command: if we have the same behaviour than before, this test would not need to exist

Reply:
Edit: After discussing with Cedric, I realized that this test is testing the values, which is not covered by the other test. So I think that you can keep it!

Reply:
actually after retesting the test I deleted in trends.ts works. But I test twice the tooltip of the GMI, it should not be that one place?

Reply:
I was going to say to keep it as you are testing the tooltip for 2 different cases:

- tooltip for when HBA1C has not been computed (stat is empty)
- tooltip for when HBA1C is computed

BUT I see that the tooltip value is the same in both cases (in your tests)... This seems wrong, don't you think?

Reply:
I see but before the migration it doesn't matter if the GMI is a value, the description didn't change so do I change the paragraph or even delete the tooltip if the GMI has a value?

Reply:
yes the information in the tooltip is the same with or without value

Reply:
Is it only on your branch only? Or is it also on dblp?